llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -3030,6 +3031,15 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
     return selectExtInst(ResVReg, ResType, I, CL::fract, GL::Fract);
   case Intrinsic::spv_normalize:
     return selectExtInst(ResVReg, ResType, I, CL::normalize, GL::Normalize);
+  case Intrinsic::spv_reflect:
+    if (!STI.canUseExtInstSet(SPIRV::InstructionSet::InstructionSet::GLSL_std_450)) {
```

Comment:
I was hoping to put this conditional in the GL overloaded version of selectExtInst so that other GL only extention instructions could benefit.

Reply:
Should I add a similar conditional to the OpenCL overloaded version of selectExtInst? Just to keep the OpenCL and GLSL functions similar.

Reply:
Not a bad idea, but in another pr. so that the changes here are more focused to the reflect feature. Feel free to file an issue and identify some tests to add.

Reply:
I made an issue for this here #123847